lib/squiddy/git_client.rb

```diff
@@ -76,21 +76,22 @@ def comment
     private
 
     def commit_title
-      "PR no. #{pr_number} merged"
+      "PR #{pr_number} merged"
     end
 
     def commit_message
       <<~MESSAGE
         #{commit_title}
 
         Squiddy-bot has merged the branch #{branch}
-        as requested by #{comment_author} in the PR number #{pr_number}.
+        into #{base_branch} after rebase as requested by #{comment_author}
+        in the PR number #{pr_number}.
```

Comment:
If you precede the PR number with a hash (eg `in the PR number ##{pr_number}.`, then I think GitHub will turn it into a link to the PR when viewing the commit on GitHub. The same probably applies to the commit_title too

Reply:
💥 

Thanks, @jamiecobbett, I'll try it out :)